sonaranalyzer-dotnet/src/SonarAnalyzer.CFG/ControlFlowGraph/BlockIdProvider.cs

```diff
@@ -0,0 +1,20 @@
+﻿/*
```

Comment:
Quite a lot of files (Blocks, and ShimLayers) are marked as removed and added. It would be nice if we can squash the commits the see them as "moved" instead